Spring-JMS/target/classes/application.properties

```diff
@@ -0,0 +1,134 @@
+# (c) Copyright IBM Corporation 2021
```

Comment:
Nothing in target should be included in this repo. Please remove this file from the pull request.